dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -590,8 +590,10 @@ public ListenableFuture<List<RouteFailure>> copy(List<Route> routes, UfsReadOpti
     List<RouteFailure> errors = Collections.synchronizedList(new ArrayList<>());
 
     for (Route route : routes) {
-      UnderFileSystem srcUfs = getUnderFileSystem(route.getSrc());
-      UnderFileSystem dstUfs = getUnderFileSystem(route.getDst());
+      UnderFileSystem srcUfs = mUfsManager.getOrAdd(new AlluxioURI(route.getSrc()),
```

Comment:
do the same thing for move and load.

Reply:
for move we want to replace it with another implementation in EE, load has its own implementation

Reply:
you mean move? I see.